### DISCUSSION TITLE
Updated MacBook last night to Sequoia and now my c++ files can't find #include and <iostream>

### DISCUSSION THREAD

**Discussion by Josephadam** [microsoft/vscode-discussions#2223] [Extension Show and Tell]
Did the new update mess with the configuration? I've tried which gcc and it's all there and g++ main and it says error can't find iostream.

**Comment by gjsjohnmurray** [comment]
Unless you are developing extensions for VS Code this is not the right place. Please read https://github.com/microsoft/vscode-discussions/discussions/5 for pointers.
